src/utils.js

```diff
@@ -186,21 +186,31 @@ function gDriveParse (catalog, tabs) {
             isCover = true
             text = text.replace(reCover, '')
           }
+
           const regDate = /\(([a-zA-Z ]*\d{4})\)/gim
           const dateMatch = regDate.exec(text)
+          const regCount = /\((count )(\d+)\)/gim
```

Comment:
Ah sure. I figured it'd be easier to force the structure because I didn't want to pick up other things and have there be a false count for a colorway. I think my initial search was finding some non-count related things. I trust you on the regex :)